frontend/src/components/HostSignupForm.tsx

```diff
@@ -0,0 +1,298 @@
+        <Divider />
+        <FormControl>
+          <FormLabel htmlFor='allowed_people'>
+            <Trans>Allowed people</Trans>
+          </FormLabel>
+          <NumberInput defaultValue={0} min={0} max={20}>
+            <NumberInputField
+              type='number'
+              id='allowed_people'
+              {...register('summary.allowed_people', { valueAsNumber: true })}
+            />
+            <NumberInputStepper>
+              <NumberIncrementStepper />
+              <NumberDecrementStepper />
+            </NumberInputStepper>
+          </NumberInput>
+        </FormControl>
+        <FormControl>
+          <CheckboxGroup onChange={(v) => restrictions.field.onChange(v)}>
+            <Box>
+              <FormLabel>
+                <Trans>Group Members</Trans>
+              </FormLabel>
+              <SimpleGrid columns={[2, null, 3]} spacing='10px'>
+                <Checkbox id='men' value='Adult men'>
+                  Adult Men
+                </Checkbox>
+                <Checkbox id='women' value='Adult women'>
+                  Adult Women
+                </Checkbox>
+                <Checkbox id='minors' value='Kids'>
+                  Minors(0-16)
+                </Checkbox>
+              </SimpleGrid>
+            </Box>
+            <Box pt='16px'>
+              <FormLabel>
+                <Trans>Pets</Trans>
+              </FormLabel>
+              <SimpleGrid columns={[2, null, 3]} spacing='10px'>
+                <Checkbox id='dogs' value='Dogs'>
+                  Pet Dog
+                </Checkbox>
+                <Checkbox id='cats' value='Cats'>
+                  Pet Cat
+                </Checkbox>
+                <Checkbox id='pets' value='Other pets'>
+                  Other Pet
+                </Checkbox>
+              </SimpleGrid>
+            </Box>
+          </CheckboxGroup>
+        </FormControl>
+        <FormControl>
+          <FormLabel htmlFor='shelter'>
+            <Trans>Shelter</Trans>
+          </FormLabel>
+          <Textarea id='shelter' {...register('summary.shelter')} />
+        </FormControl>
+        <Divider />
+        <FormControl display='flex' alignItems='center'>
+          <FormLabel htmlFor='avaliable' mb='0'>
+            <Trans>Avaliable</Trans>
+          </FormLabel>
+          <Switch
+            id='avaliable'
+            onChange={(state) => active.field.onChange(state)}
+          />
+        </FormControl>
+        <FormControl>
+          <FormLabel htmlFor='avaliability-date'>
+            <Trans>Availability</Trans>
+          </FormLabel>
+          <DatePicker
+            id='avaliability-date'
+            selected={parseDate(dateAvailable.field.value)}
+            onChange={(date) =>
+              dateAvailable.field.onChange(formatDate(date as Date))
+            }
+            disabled={!getValues('summary.availability.active')}
+            showPopperArrow={true}
+            dateFormat='dd-MM-yyyy'
```

Comment:
Just to make sure -- the API format is YYYY-MM-DD (I think). Can you make sure that's what we send back to the server?